docs/make.jl

```diff
@@ -7,40 +7,106 @@ project_path = Base.Filesystem.joinpath(Base.Filesystem.dirname(Base.source_path
 Pkg.develop(; path=project_path)
 
 using Documenter
+using Literate
+using DocumenterInterLinks
+using DocumenterCitations
+
 using QEDbase
+using QEDcore
+using QEDprocesses
 
-using DocumenterCitations
+# some paths for links
+readme_path = joinpath(project_path, "README.md")
+index_path = joinpath(project_path, "docs/src/index.md")
+license_path = "https://github.com/QEDjl-project/QEDbase.jl/blob/main/LICENSE"
+
+# Copy README.md from the project base folder and use it as the start page
+open(readme_path, "r") do readme_in
+    readme_string = read(readme_in, String)
+
+    # replace relative links in the README.md
+    readme_string = replace(readme_string, "[MIT](LICENSE)" => "[MIT]($(license_path))")
+
+    open(index_path, "w") do readme_out
+        write(readme_out, readme_string)
+    end
+end
+
+# setup interlinks
+links = InterLinks("QEDcore" => "https://qedjl-project.github.io/QEDcore.jl/dev/")
 
+# setup Bibliography
 bib = CitationBibliography(joinpath(@__DIR__, "Bibliography.bib"))
 
+# setup examples using Literate.jl
+literate_paths = [
+    Base.Filesystem.joinpath(project_path, "docs/src/tutorial/lorentz_vectors.jl"),
+    Base.Filesystem.joinpath(project_path, "docs/src/tutorial/particle.jl"),
+    Base.Filesystem.joinpath(project_path, "docs/src/tutorial/particle_stateful.jl"),
+    Base.Filesystem.joinpath(project_path, "docs/src/tutorial/model.jl"),
+    Base.Filesystem.joinpath(project_path, "docs/src/tutorial/process.jl"),
+    Base.Filesystem.joinpath(project_path, "docs/src/tutorial/phase_space_point.jl"),
+]
+
+tutorial_output_dir = mktempdir(joinpath(project_path, "docs/src/"); cleanup=true)
```

Comment:
While I like the idea, there is a problem with it: The generated files will live in a directory that is named differently across different builds, which means the URI will be different as well. This means that external links referencing these tutorials would have to be updated with every push to the `gh-pages` branch, or the links would be invalid. 

It doesn't seem like there is a parameter for `mktempdir` to provide a directory name though...

Reply:
You are right, of course. In 7a51195, I changed it to a fixed name `generated` and handled the garbage collections manually. Maybe we should open an issue on Julia to extend the functionality of `mktempdir` to allow fixed names.